Updates (#478)
digital-asset/docs.daml.com: docs/2.8.0/docs/query/pqs-user-guide.rst

   the oldest transaction that is unaffected by the pruning operation.

The following data will be deleted:

-  Transactions with offsets up to and including the target offset.
-  Events, archived contracts and exercise payloads associated with the
   deleted transactions.

The following data will be unaffected: 

- Transaction related data (event, choices, or contracts) for transaction with
  an offset that is greater than the effective pruning target offset.

Pruning is a destructive operation and cannot be undone. If necessary,
make sure to backup your data before performing any pruning operations.

There are some constraints when using either pruning method:

1. The provided target offset must be within the bounds of the
   contiguous history. If the target offset is outside the bounds, an
   error will be raised.
2. The pruning operation cannot coincide with the latest consistent
   checkpoint of the contiguous history. If it does, an error will be
   raised.

Pruning with PQS CLI
-----------------------

The PQS CLI provides a ``prune`` command that allows you to prune the
ledger data up to a specified offset, timestamp, or duration.

> For detailed information on all available options, please run
> ``./PQS.jar datastore postgres-document prune --help-verbose``.

Reviewer: ```suggestion
For detailed information on all available options, run
``./PQS.jar datastore postgres-document prune --help-verbose``.
```

> To use the ``prune`` command, you need to provide a pruning target as an

Reviewer: ```suggestion
To use the ``prune`` command, provide a pruning target as an
```

argument. The pruning target can be an offset, a timestamp (ISO 8601),
or a duration (ISO 8601):

::

   ./PQS.jar datastore postgres-document prune --prune-target <pruning_target>

> By default, the ``prune`` command performs a dry run, which means it
> will only display the effects of the pruning operation without actually
> deleting any data. To execute the pruning operation, you need to add the
> ``--prune-mode Force`` option:

Reviewer: ```suggestion
By default, the ``prune`` command performs a dry run: it
displays only the effects of the pruning operation, without actually
deleting any data. To execute the pruning operation, add the
``--prune-mode Force`` option:
```

::

   ./PQS.jar datastore postgres-document prune --prune-target <pruning_target> --prune-mode Force

> In addition to providing an offset as ``--prune-target``, a timestamp
> or duration can also be used as pruning cut-off. For example, to prune
> data older than 30 days (relative to now), you can use the following
> command:

Reviewer: ```suggestion
Instead of providing an offset as the ``--prune-target``, you can use a timestamp
or duration as the pruning cutoff. For example, the following command prunes
data older than 30 days (relative to now):
```

::

   ./PQS.jar datastore postgres-document prune --prune-target P30D

> To prune data up to a specific timestamp, you can use the following
> command:

Reviewer: ```suggestion
The following example prunes data up to a specific timestamp:
```

::

   ./PQS.jar datastore postgres-document prune --prune-target 2023-01-30T00:00:00.000Z

Pruning with ``prune_to_offset``
--------------------------------

> The ``prune_to_offset`` function is a PostgreSQL function that allows
> you to prune the ledger data up to a specified offset. It has the same
> behavior as the ``datastore postgres-document prune`` command, but does
> not feature a dry-run option.

Reviewer: ```suggestion
The ``prune_to_offset`` PostgreSQL function allows
you to prune ledger data up to a specified offset. It has the same
behavior as the ``datastore postgres-document prune`` command, except it does not
offer dry runs.
```

> To use ``prune_to_offset``, you need to provide a offset as a text
> argument:

Reviewer: ```suggestion
To use ``prune_to_offset``, provide an offset as a text
argument:
```

.. code:: sql

   SELECT * FROM prune_to_offset('<offset>');

> The function will delete transactions and update active contracts as
> described above.
> 
> You can use ``prune_to_offset`` in combination with the ``get_offset``
> function to prune data up to a specific timestamp or interval. For
> example, to prune data older than 30 days, you can use the following
> query:

Reviewer: ```suggestion
This function deletes transactions and update active contracts as
described :ref:`earlier in this section <pqs-pruning-behavior>`.

To prune data up to a specific timestamp or interval, use ``prune_to_offset`` 
in combination with the ``get_offset`` function. For example, the following 
query prunes data older than 30 days:
```
